scripts/compatibility-test/splunk/s3_v4_tests.txt

```diff
@@ -129,4 +129,6 @@ s3tests.functional.test_s3:test_ranged_request_response_code
 s3tests.functional.test_s3:test_ranged_request_skip_leading_bytes_response_code 
 s3tests.functional.test_s3:test_ranged_request_return_trailing_bytes_response_code 
 s3tests.functional.test_s3:test_ranged_request_invalid_range 
-s3tests.functional.test_utils:test_generate 
```

Comment:
test_generate should be last one @sali619

Reply:
@nileshgovande : Edited the v4 txt file and added testgenerate a eof.